From 94f5feb90199427b59ddf524ccad74b2512ef68a Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Wed, 7 Feb 2024 18:32:27 +0100
Subject: libstdc++: avoid <memory> include by assigning nullptr in
 token_managed_profile_creation_delegate.h

In libstdc++ implementation of std::unique_ptr, it requires
definition to assign to nullptr. As the initial unique_ptr value is
already nullptr, just remove the assignation.

Bug: 41455655
Change-Id: I7f5e4b7a527f1f8be42500a21108c1c78c41bcc6
---
 .../enterprise/signin/token_managed_profile_creation_delegate.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/enterprise/signin/token_managed_profile_creation_delegate.h b/chrome/browser/enterprise/signin/token_managed_profile_creation_delegate.h
index b5b7719a07e7f..15ef7dcb4b8e1 100644
--- a/chrome/browser/enterprise/signin/token_managed_profile_creation_delegate.h
+++ b/chrome/browser/enterprise/signin/token_managed_profile_creation_delegate.h
@@ -33,7 +33,7 @@ class TokenManagedProfileCreationDelegate
 
  private:
   const std::string enrollment_token_;
-  std::unique_ptr<signin_util::CookiesMover> cookies_mover_ = nullptr;
+  std::unique_ptr<signin_util::CookiesMover> cookies_mover_;
 };
 
 #endif  // CHROME_BROWSER_ENTERPRISE_SIGNIN_TOKEN_MANAGED_PROFILE_CREATION_DELEGATE_H_
-- 
2.40.1

